lib/terraforming/resource/sns_topic_subscription.rb

```diff
@@ -0,0 +1,83 @@
+module Terraforming
+  module Resource
+    class SNSTopicSubscription
+      include Terraforming::Util
+
+      def self.tf(client: Aws::SNS::Client.new)
+        self.new(client).tf
+      end
+
+      def self.tfstate(client: Aws::SNS::Client.new)
+        self.new(client).tfstate
+      end
+
+      def initialize(client)
+        @client = client
+      end
+
+      def tf
+        apply_template(@client, "tf/sns_topic_subscription")
+      end
+
+      def tfstate
+        subscriptions.reject{|x| x["Protocol"].include?("email")}
```

Comment:
__[rubocop]__
* Space missing to the left of {.
* Space between { and | missing.
* Space missing inside }.

<!-- Comment by [SideCI](https://sideci.com) -->